### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 _Keyboard: "Results table" is not accessible using tab key and arrow keys

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17452]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Sep2022;#mssql for VSCode;#Win32;#DesktopApp;#A11ySev1;#Keyboard;#AINotInScope;#WCAG2.1.1;#Win11;#Closed;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Visual studio code-insiders Version: 1.72.0-insider (user setup)

## Repro Steps:

1. **Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix**
2. Open Visual studio code
3. Tab till extension button and search for "mssql"
4. Tab till install and move to the SQL Server button
5. Tab till Add Connection Button and insert the valid credentials
6. After adding the server move to database expand it -->System database expand-->Master expand-->Tables expand -->System table expand and right click on it and select top 1000 option
7. Select top 1000 window will open
8. Execute the Query
9. Observe that "Results table" is not accessible using tab key and arrow keys

## Actual:
"Results table" is not accessible using tab key and arrow keys. Increasing/decreasing the size of the cell and scrolling function of the table are not accessible with the keyboard

## Expected:
Keyboard focus should move to the column headers and should be able to access the Scrolling function of the table using arrow keys and also should be able to increase or decrease the cell size using keyboard

## User Impact:
Keyboard users are unable to access the whole content in the table and also not able to increase or decrease the size of the table cells and scrolling function of the table

## Attachment:

![17452_A11y_mssql for VSCode_Select top 1000 _Keyboard](https://user-images.githubusercontent.com/98821792/191478376-76a4bdfb-e7d9-44bc-8f28-f99ae34d1842.gif)

![17451_closed](https://user-images.githubusercontent.com/98821792/203248144-302820bc-075b-4df4-ac63-bc2ddcc836fb.gif)

**Comment by prabhudevu** [comment]
I have verified the issue the Issue in the latest version of the app MSSQL 1.17.0. Issue is fixed, hence closing the bug

**Comment by prabhudevu** [comment]
Verified the issue in the latest Environment of MSSQL 1.1.7.0 issue is fixed, hence closing the bug. Added attachment for reference
